portal/src/components/DoDontExample/DoDontExample.tsx

```diff
@@ -30,7 +30,9 @@ export const DoDontExample: React.FC<Props> = ({ type, content, image = "", desc
             {content ? (
                 <div>{content}</div>
             ) : (
-                <img className="jkl-portal-do-dont-example__image" src={withPrefix(image)} alt={altText} />
+                <div className="jkl-portal-do-dont-example__image">
+                    <PortalImg src={image} alt={altText} />
+                </div>
```

Comment:
Er det nødvendig med denne funksjonaliteten? Hvis den skal med må det i hvert fall tweakes en del: nå blir det seende ganske rart ut pga marginer/padding på `jkl-portal-image`, og teksten under virker litt malplassert.

Reply:
![image](https://user-images.githubusercontent.com/46530368/90748861-02224a80-e2d3-11ea-919b-8a3e7e7cdc6e.png)
Sånn da?

Reply:
Ser veldig mye bedre ut i hvert fall! Synes fortsatt "Klikk..."-teksten er litt forstyrrende, men ikke like ille nå som bildet er på plass 😅

Reply:
Tok vekk klikk for større teksten på doDont, og fikset spacing på desktop